<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nombrelbl">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nombreEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="correolbl">
          <property name="text">
           <string>Correo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="correoEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="telefonolbl">
          <property name="text">
           <string>Telefono:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="telefonoEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="comidaDialbl">
          <property name="text">
           <string>Usted prefiere ir a un restaurante a:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comidacomboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="dialbl">
          <property name="text">
           <string>Y acostumbra a ir el día:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="diacomboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="comidalbl">
        <property name="text">
         <string>Comida preferida:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="comidaEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="informacionlbl">
        <property name="text">
         <string>Ver Información:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="InformacionText"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="enviarButton">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="verButton">
          <property name="text">
           <string>Ver Información</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
